dev/com.ibm.ws.rest.handler/bnd.bnd

```diff
@@ -20,9 +20,6 @@ Web-ContextPath: ibm/api
 IBM-Authorization-Roles: com.ibm.ws.management
 IBM-Web-Extension-Processing-Disabled: true
 
-Import-Package: \
-    org.apache.commons.fileupload*;version=1.2.1;packageType="was_internal", \
```

Comment:
same as above, not 100% sure hte impact. seems like maybe a holdover from a previous version of WAS?

Reply:
Same answer as above